WebContent/WOH-addRestaurant.jsp

```diff
@@ -87,8 +87,9 @@
 						
 					<!-- Notes Text Area -->
 						<div class="form-group">
-  							<label for="WOH-addRestaurants-notesInput">Notes</label>
-  							<textarea class="form-control WOH-addRestaurants-notesInput" rows="5" name="notes"></textarea>
+							<input type="hidden" name="notes" class="WOH-addRestaurant-notesInput">
+  							<label for="WOH-addRestaurants-notesTextArea">Notes</label>
+  							<textarea class="form-control WOH-addRestaurants-notesTextArea" rows="5" name="notesInput"></textarea>
```

Comment:
For right now, this is okay, but later, I want these notes to go into a table so it looks more like comments on Facebook or something like that.  this will allow the user to easily edit and delete notes.